test

```diff
@@ -96,22 +96,40 @@ function cov_pass {
 
 	mkdir -p "$COVERDIR"
 
-	# PKGS_DELIM contains all the core etcd pkgs delimited by ',' which will be profiled for code coverage.
-	# Integration tests will generate code coverage for those pkgs 
+	# convert COVERDIR to absolute path
+	# e2e needs absolute path to put its coverfiles
```

Comment:
should probably accept relative paths as well... use `filepath.IsAbs` in the e2e code to check for absolute; if it isn't, `filepath.Join("..", path)`